about.html

```diff
@@ -0,0 +1,77 @@
+            <li class="nav-link"><a href="#about-main">Experience</a></li>
+            <li class="nav-link"><a href="#about-more">What's On</a></li>
+            <li class="nav-link"><a href="#about-more">Awards</a></li>
+            <li class="nav-link"><a href="">Contact us</a></li>
+          </ul>
+        </div>
+        <div class="nav-end">
+          <button id="tickets-btn">Tickets</button>
+        </div>
+        <div class="menu-button-wrap">
+            <div class="menu-line"></div>
+        </div>
+      </nav>
+      <section class="about-heading">
+        <h1 class="about-main-heading">The Harry Potter film series</h1>
+        <p class="about-description">The series made Leavesden its home for more than ten years. As the books were still being released while the films were being made, the production crew saved many of the iconic sets, props and costumes that were created especially for the films.</p>
+        <div class="about-contact">
+          <p class="contact-text">If you have questions about your upcoming visit, there are many ways we can help!</p>
+          <p class="contact-email"><strong>studiotour@wbuk.org</strong></p>
+        </div>
+      </section>
+  </header>
+  <main>
+    <section id="about-main" class="main-about">
+      <h3 class="main-about-heading">HARRY POTTER AND THE DEATHLY HALLOWS – PART 1 AND 2</h3>
+      <div id="about-heading-under"></div>
+      <p class="about-details">Harry Potter and the Deathly Hallows – Part 1 began filming on Thursday 19th February 2009, and the second installment, Harry Potter and the Deathly Hallows – Part 2, was filmed back-to-back with Part 1. Five 32-ton trucks worth of polystyrene were needed, to create all the rubble for the destruction of Hogwarts at the end of the final film.</p>
+      <div class="about-main-img"></div>
+    </section>
+    <section id="about-more" class="more-about">
+      <h3 class="more-about-heading">More in the Series</h3>
+      <div id="about-heading-under"></div>
+      <p class="more-about-details">Over the ten years, an incredible 588 sets were created at Leavesden Studios.</p>
+      <div class="about-other">
+        <div class="other-left">
+          <h4 class="about-other-inner-heading">2000</h4>
+          <p class="about-other-inner-text">The Philosopher’s Stone</p>
+        </div>
+        <div class="other-right">
+            <h4 class="about-other-inner-heading">2006</h4>
+            <p class="about-other-inner-text">The Order of the Phoenix</p>
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-logo">
+        <img src="./media/Warner-Bros-Logo-1.png" alt="" class="logo-img-footer">
+      </div>
+      <div class="footer-details">
+        <p class="footer-details-top">‘Wizarding World’ and all related names, characters and indicia are trademarks of and © Warner Bros. Entertainment Inc.</p>
+        <p class="footer-details-bottom">© J.K. Rowling</p> 
+      </div>
+  </footer>
```

Comment:
- Optional
The footer for the about page has a different background color. Currently, your `main` section and `footer` section have the same bg color which makes them seem like a single section. I am marking this as optional because the first reviewer didn't point this out. However, it is an integral part of the interface and in my opinion, you should absolutely implement it. I am sure it would look much better that way.

| Your design | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/204079118-8cabeabb-7d7d-4177-b14a-f8900e110aa1.png) | ![image](https://user-images.githubusercontent.com/33268927/204079097-50a9c02e-3c71-4461-b23b-c975b8a52b09.png) |